packages/ui-app/src/Chart/Base.tsx

```diff
@@ -0,0 +1,37 @@
+// Copyright 2017-2019 @polkadot/ui-app authors & contributors
+// This software may be modified and distributed under the terms
+// of the Apache-2.0 license. See the LICENSE file for details.
+
+import { BareProps } from '../types';
+
+import React from 'react';
+import styled from 'styled-components';
+
+import { classes } from '../util';
+
+type Props = BareProps & {
+  children: React.ReactNode
+};
+
+const Wrapper = styled.div`
+  position: relative;
+  display: inline-block;
+  padding: 1em 1em 0;
+  height: 15vw;
+  width: 15vw;
+`;
+
+export default class BaseChart extends React.PureComponent<Props> {
```

Comment:
Keeping the base wrapper separate (if we add multiple chart types, as was the intent)

Reply:
`Wrapper` :) capitals...